projects/core/src/lib/backend/backend.model.ts

```diff
@@ -274,10 +274,12 @@ export interface OperatorParams {
     [key: string]: ParamTypes;
 }
 
+export type NamedData = string;
```

Comment:
would a `NamedDataDict` make sense here? Which would then be parsed into a NamedData class with namespace and name etc. s.t. we can render it better?